msctl

```diff
@@ -276,6 +276,9 @@ mscs_defaults() {
 ; for the world server selected.
 # mscs-default-server-command=\$JAVA -Xms\$INITIAL_MEMORY -Xmx\$MAXIMUM_MEMORY \$JVM_ARGS -jar \$SERVER_LOCATION/\$SERVER_JAR \$SERVER_ARGS
 
+; Default behavior if to restart the server after crash is detected (default disabled).
+# mscs-default-restart-after-crash=false
```

Comment:
`true/false` is not used for any other mscs setting, will using `0/1` still evaluate to `true/false`?

Reply:
Good point. We probabably should add code like:

```bash
#!/bin/sh

true_value() {
  local VALUE
  VALUE=$(echo "$1" | tr '[:upper:]' '[:lower:]')
  if [ "$VALUE" = "true" ] || [ $VALUE = 1 ]; then
    return 0
  fi
  if [ "$VALUE" = "false" ] || [ $VALUE = 0 ]; then
    return 1
  fi
  return -1
}

if true_value "true"; then echo "true"; else echo "false?"; fi     # Prints true
if true_value "TRUE"; then echo "true"; else echo "false?"; fi     # Prints true
if true_value 1; then echo "true"; else echo "false?"; fi          # Prints true
if true_value "1"; then echo "true"; else echo "false?"; fi        # Prints true


if ! true_value "false"; then echo "false"; else echo "true?"; fi  # Prints false
if ! true_value "FALSE"; then echo "false"; else echo "true?"; fi  # Prints false
if ! true_value 0; then echo "false"; else echo "true?"; fi        # Prints false
if ! true_value "0"; then echo "false"; else echo "true?"; fi      # Prints false

if true_value "cat"; then echo "cat?"; else echo "not cat?";  fi   # Prints error message
```

Reply:
Very nice, might be easier for users to be able to use `true/false` over `1/0`

Reply:
Do you think expanding this to accept `on/off` would be useful?

Reply:
Or heck, even `yes/no`

I better stop before going off the rails...

Reply:
That is doable...

```bash
true_value() {
  local VALUE
  VALUE=$(echo "$1" | tr '[:upper:]' '[:lower:]')
  if [ $VALUE = 1 ] || [ "$VALUE" = "true" ] || [ "$VALUE" = "yes" ] || [ "$VALUE" = "on" ]; then
    # Return a true value (success).
    return 0
  fi
  if [ $VALUE = 0 ] || [ "$VALUE" = "false" ] || [ "$VALUE" = "no" ] || [ "$VALUE" = "off" ]; then
    # Return a false value (failure).
    return 1
  fi
  return -1
}
```

Reply:
I just created PR #274 to add this `true_value` code

Reply:
thanks for this!